packages/govuk-frontend-review/tasks/watch.mjs

```diff
@@ -43,7 +43,7 @@ function getTasks(options) {
     ),
     'compile:scss watch': () =>
       gulp.watch(
-        ['**/*.scss', join(paths.package, 'dist/govuk/all.scss')],
+        ['**/*.scss', join(paths.package, 'dist/govuk.scss')],
```

Comment:
**question/issue** Should this be `dist/govuk/index.scss`, I don't think the `dist/govuk.scss` file exists?

Reply:
Whoops! 😅